libfabric.spec

```
@@ -1,10 +1,8 @@
 %define suse_libname libfabric1
-#global dl_version 1.11.1rc1
-
-%{?dl_version:%global autosetup_args --p1 -n libfabric-%{dl_version}}
+%global dl_version 1.12.0rc1
```

We're starting to gravitate towards some standards that allow us to easily switch between _pre_ and _GA_ releases of packages without having to change lots of things all over the spec file.  First we start with defining the components of the version and build the `%dl_version` from it:

```suggestion
%global major 1
%global minor 12
%global bugrelease 0
%global prerelease rc1
%global dl_version  %{major}.%{minor}.%{bugrelease}%{?prerelease:%{prerelease}}
```